### PR TITLE
add Deco recipe

### PR DESCRIPTION
**Pull request by Enhex**
Specify library name and version:  **deco/1.0.0**

This is a library I made.

---

- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

File: recipes/deco/all/conanfile.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
+from conans import ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+class DecoConan(ConanFile):
+    name = "deco"
+    license = "Apache 2.0 with LLVM exceptions"
```
Comment (uilianries) [inline recipes/deco/all/conanfile.py:7]: What's the official SPDX name for it?

File: recipes/deco/all/conanfile.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    description = "Delimiter Collision Free Format"
+    topics = ("serialization")
+    homepage = "https://github.com/Enhex/Deco"
+    url = "https://github.com/conan-io/conan-center-index"
+    no_copy_source = True
+    settings = ("compiler")
```
Comment (uilianries) [inline recipes/deco/all/conanfile.py:13]:
```suggestion
    settings = ("compiler", "build_type", "os", "arch")
```

File: recipes/deco/all/conanfile.py (hunk 1 above)
Comment (Enhex) [reply recipes/deco/all/conanfile.py:7]: following "SPDX License Exceptions" in https://spdx.dev/ids/
and https://spdx.org/licenses/LLVM-exception.html
it should be `Apache-2.0 WITH LLVM-exception`

Comment (uilianries) [reply recipes/deco/all/conanfile.py]: So, please, use `LLVM-exception` as described as official short name. It makes easier to parse and find when using scripts. Also, white space is identified as license separator, so it's not allowed.

Comment (Enhex) [reply recipes/deco/all/conanfile.py]: already updated

File: recipes/deco/all/conanfile.py [hunk 3]
```diff
@@ -0,0 +1,56 @@
+from conans import ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+class DecoConan(ConanFile):
+    name = "deco"
+    license = "Apache-2.0 WITH LLVM-exception"
```
Comment (uilianries) [inline recipes/deco/all/conanfile.py:7]:
```suggestion
    license = "Apache-2.0-WITH-LLVM-exception"
```

white space are not allowed

**Review by SSE4 — APPROVED** [review]

**Review by uilianries — APPROVED** [review]

**Comment by Enhex** [comment]
missing a reviewer?

**Comment by uilianries** [comment]
Up!

**Comment by Enhex** [comment]
> Up!

I'm not sure what you mean

**Comment by uilianries** [comment]
@Enhex By default, Github sorts the PR/issues list by last updated comment. So when I wrote Up! I forced this PR to the top of pulls list, so any reviewer will see it at first sight.

**Review by prince-chrismc — APPROVED** [review]

**Comment by prince-chrismc** [comment]
> By default, Github sorts the PR/issues list by last updated comment.

Hmmm, my default setting is newest 

image

**Pull request merged**